<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageCaptureWidgetClass</class>
 <widget class="QWidget" name="ImageCaptureWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageCaptureWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="imgidLabel">
     <property name="text">
      <string>ID：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="previewLabel">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="captureBtn">
       <property name="text">
        <string>截取图像</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
